### PR TITLE
OSSMDOC-213: Update existing federation reference.

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Deleted one line and revised another based on story description. Changes were made for the v2 file. v1 was left unchanged.

Cherry pick to 4.6, 4.7, 4.8, and 4.9

@JStickler please let me know if you find anything I need to revise. Thanks! 

Preview link: https://deploy-preview-35770--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/preparing-ossm-installation.html#ossm-supported-configurations_preparing-ossm-installation

### CODE REVIEW

**Review by otikhomi — APPROVED** [review]

**Comment by JStickler** [comment]
@rcernich
>  The deployment must be contained to a single {product-title} cluster that is not federated.

Should this be 
" * The deployment must be contained to a single {product-title} cluster."

Or is it correct to delete the entire line for 2.1.

**Review by rcernich — APPROVED** [review]

File: modules/ossm-supported-configurations.adoc [hunk 1]
```diff
@@ -16,11 +16,10 @@ The following configurations are supported for the current release of {ProductNa
 Red Hat OpenShift Online is not supported for {ProductName}.
 ====
 
-* The deployment must be contained to a single {product-title} cluster that is not federated.
 * This release of {ProductName} is only available on {product-title} x86_64, IBM Z, and IBM Power Systems.
 ** IBM Z is only supported on {product-title} 4.6 and later.
 ** IBM Power Systems is only supported on {product-title} 4.6 and later.
-* Configurations where all {ProductShortName} components are contained in the {product-title} cluster in which it operates. {ProductName} does not support management of microservices that reside outside of the cluster, or in a multi-cluster scenario.
+* Configurations where all {ProductShortName} components are contained in the {product-title} cluster in which the {ProductShortName} operates. {ProductName} does not support management of microservices that reside outside of the cluster.
```
Comment (rcernich) [inline modules/ossm-supported-configurations.adoc:22]: Maybe it's simpler to say something like:
```
 * Configurations where all {ProductShortName} components are contained within a single {product-title} cluster.  {ProductName} does not support management of microservices that reside outside the cluster within which {ProductShortName} is running.
```

Comment (rh-tokeefe) [reply modules/ossm-supported-configurations.adoc]: @rcernich the change has been submitted.

**Pull request merged**